packages/esm-patient-forms-app/src/hooks/use-forms.ts

```diff
@@ -67,3 +67,5 @@ function mapToFormCompletedInfo(
     };
   });
 }
+
+const removeComponentForm = (formName: string): boolean => !formName.toLowerCase().includes('component');
```

Comment:
This function name doesn't accurately describe what it does. It would be better called something like 'isNotComponent'.
However, for boolean values it's better to put it in the positive, so it would be better format the function as 'isComponent', then do the negation on line 12.